c2pa_c_ffi/src/c_api.rs

```diff
@@ -1729,6 +1846,122 @@ mod tests {
         }
     }
 
+    #[test]
+    fn builder_create_intend_and_sign() {
```

Comment:
Did you intend to use intend rather than intent? Or was your intent to use intent?
Sorry I just had to write that sentence ;)